src/index.js

```diff
@@ -1,10 +1,21 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
-import App from './App'
+import { BrowserRouter as Router } from 'react-router-dom'
+
 import * as serviceWorker from './serviceWorker'
+import App from './App'
+import Routes from './routes'
 
 // eslint-disable-next-line react/no-render-return-value
-const render = () => ReactDOM.render(<App />, document.getElementById('root'))
+const render = () => {
+  ReactDOM.render(
+    <Router>
+      <App />
+      <Routes />
+    </Router>,
```

Comment:
let's move this to App.js to have here just ReactDOM.render(<App />)

Reply:
DONE!